lib/sqlalchemy/sql/selectable.py

```diff
@@ -1995,7 +2020,7 @@ class CTE(
     __visit_name__ = "cte"
 
     _traverse_internals: _TraverseInternalsType = (
```

Comment:
**mike bayer** (zzzeek) wrote:

remove

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4325